js/chirp.js

```diff
@@ -1,7 +1,15 @@
 function chirp(n){
-  // FIX ME
+  //base case
+  console.log(n);
+  if( n === 1){
+    return ' chirp';
+  } else {
```

Comment:
Because of the return in the if statement, you really don't need the else part of this code.